apps/ledger-live-desktop/src/renderer/modals/SignMessage/steps/StepSummary.tsx

```diff
@@ -175,7 +226,8 @@ export function StepSummaryFooter({ transitionTo }: StepProps) {
         onClick={() => {
           transitionTo("sign");
         }}
-        primary
+        style={{ borderRadius: 20, backgroundColor: "white" }}
+        color="neutral.c00"
```

Comment:
You cannot change the footer button only for this step while the other steps keep the old design, it would be inconsistent across the flow

Not sure why the ticket asks to update the design on a single step here, it feels wrong and should have been picked up when refining the ticket
IMO you do it totally or you don't